src/lib/components/Sidebar/Sidebar.stories.tsx

```diff
@@ -1,23 +1,26 @@
-import { Meta, Story } from '@storybook/react/types-6-0';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
 import { BiBuoy } from 'react-icons/bi';
 import { HiChartPie, HiViewBoards, HiInbox, HiUser, HiShoppingBag, HiArrowSmRight, HiTable } from 'react-icons/hi';
-
-import { Sidebar } from '.';
+import { Sidebar, SidebarComponent } from '.';
 import { Badge } from '../Badge';
 import { Button } from '../Button';
+import { SidebarContext } from './SidebarContext';
 
 export default {
   title: 'Components/Sidebar',
-  component: Sidebar,
-} as Meta;
+  component: SidebarComponent,
+} as ComponentMeta<typeof SidebarComponent>;
 
-const Template: Story = (args) => <Sidebar {...args} />;
+const Template: ComponentStory<typeof SidebarComponent> = (args) => (
+  <SidebarContext.Provider value={{ collapsed: args.collapsed ?? false }}>
+    <SidebarComponent className="px-0" {...args} />
+  </SidebarContext.Provider>
+);
 
-export const DefaultSidebar = Template.bind({});
-DefaultSidebar.storyName = 'Default';
-DefaultSidebar.args = {
+export const Default = Template.bind({});
+Default.args = {
   children: (
-    <Sidebar collapsed={false}>
+    <SidebarComponent>
```

Comment:
I'm very confuse about this `SidebarComponent`... isn't just `Sidebar`? Why?

Reply:
Yea, agreed.

Reply:
The reason this is exported is to accommodate a strange issue with Storybook and React Context. I'm looking for a solution.